source/style.css

```diff
@@ -0,0 +1,288 @@
+header {
+    position: relative;
+    left: 0;
+    z-index: 0;
+}
+
+header:before {
+    content: "";
+    position: fixed;
+    left: 0;
+    right: 0;
+    z-index: -1;
+    display: block;
+    background-image: url("../images/headerImage.jpg");
+    background-size: cover;
+    filter: blur(3px);
+    max-width: 100%;
+    height: 340px;
+    padding: 100px 200px;
+}
+
+.top-container {
+    display: flex;
+    justify-content: space-between;
+    position: relative;
+    padding: 55px 220px;
+}
+
+.logo-container {
+    display: flex;
+}
+
+.logo {
+    background-color: transparent;
+    position: relative;
+    width: 100px;
+    vertical-align: middle;
+
+}
+.logo-name {
+    color: white;
+    display: flex;
+    align-items: flex-start;
+    padding: 10px 0;
+    font-weight: 800;
+    font-family: 'Montserrat', sans-serif;
+}
+
+.navigation-top {
+    display: flex;
+    justify-content: flex-end;
+}
+
+a {
+    text-decoration: none;
+    color: white;
+    background-color: transparent;
+}
+
+li {
+    font-family: 'Montserrat', sans-serif;
+    font-weight: 400;
+    font-size: large;
+    margin-left: 1.5em;
+    padding: 0 20px;
+    color: rgb(255, 102, 0);
+}
+
+li:first-child{
+    list-style-type: none;
+}
+
+h1 {
+    color: white;
+    display: flex;
+    justify-content: center;
+    font-family: 'Montserrat', sans-serif;
+    font-size: 60px;
+}
```

Comment:
What if in another part of the page you need to have an anchor tag (<a>) but with the color red?